Formula/kubecfg.rb

```diff
@@ -0,0 +1,48 @@
+class Kubecfg < Formula
+  desc "Manage complex enterprise Kubernetes environments as code"
+  homepage "https://github.com/ksonnet/kubecfg"
+  url "https://github.com/ksonnet/kubecfg/archive/v0.8.0.tar.gz"
+  sha256 "25d054af96a817bad0f33998895a9988c187e1399822c8220528e64f56ccb3ae"
+  head "https://github.com/ksonnet/kubecfg.git"
```

Comment:
Remove head

Reply:
Really? I will comply, but I suggest that the Acceptable Formulae document should document the deprecation of `head` and any conditions that apply.